spec/views/hearings/_hearing_events.html.haml_spec.rb

```diff
@@ -68,16 +68,14 @@
           .and have_content('ended late')
       end
 
-      context 'with notes containing unicode characters' do
-        let(:hearing_event_with_unicode_notes) do
-          hearing_event_class.new(description: 'day 1 start', note: '!\"#£%&()*,-./Æ½ŵ€',
+      context 'with notes containing html, unicode and crlf_escape_sequences' do
+        let(:hearing_event_with_special_characters) do
+          hearing_event_class.new(description: 'day 1 start', note: free_text,
```

Comment:
where is `free_text` defined. clearer to have it in block or directly as string, as before

Reply:
@jsugarman, free_text is defined in the new spec/support/shared_examples_for_free_text_fields.rb. I have tried to extract the tests to a shared example so that they can be reused across view specs but that means I have to define free_text in shared_examples_for_free_text_fields.rb. Maybe that isn't clear/I shouldn't be using that approach?